src/knx/application_layer.cpp

```diff
@@ -181,32 +239,36 @@ void ApplicationLayer::dataSystemBroadcastIndication(HopCountType hopType, Prior
             popByte(testInfo[1], data + 5);
             propertyId = (propertyId >> 4) & 0x0FFF;;
             testInfo[0] &= 0x0F;
-            _bau.systemNetworkParameterReadIndication(priority, hopType, objectType, propertyId, testInfo, sizeof(testInfo));
+            //TODO: _bau.systemNetworkParameterReadLocalConfirm(priority, hopType, objectType, propertyId, testInfo, sizeof(testInfo), status);
```

Comment:
Why are these calls commented out? They should have an empty standard implementation, shouldn't they?

Reply:
As they are not used I did not implement the empty ones. Shall I do implement the empty stuff?

Reply:
BTW: you cannot see it really well from the DIFF.
Have a look here: https://github.com/nanosonde/knx/blob/refactor_broadcast/src/knx/application_layer.cpp#L80
And here: https://github.com/nanosonde/knx/blob/refactor_broadcast/src/knx/application_layer.cpp#L179

Reply:
Just added the empty confirm counterparts.